### ISSUE TITLE
Exception for nifty

### ISSUE THREAD

**Issue by cdeil** [haypo/sixer#3]
I tried running sixer on https://github.com/information-field-theory/nifty and got this error:

```
$ sixer all .
(Dry run: don't modify files)

Patch ./nifty_core.py with iteritems, long, xrange
Patch ./nifty_explicit.py with long, xrange
Patch ./nifty_power.py with xrange
ERROR while patching ./smoothing.py
Traceback (most recent call last):
  File "/Users/deil/Library/Python/3.4/bin/sixer", line 11, in <module>
    sys.exit(main())
  File "/Users/deil/Library/Python/3.4/lib/python/site-packages/sixer.py", line 1299, in main
    Patcher(operations, options).main(paths)
  File "/Users/deil/Library/Python/3.4/lib/python/site-packages/sixer.py", line 1277, in main
    self.patch(filename)
  File "/Users/deil/Library/Python/3.4/lib/python/site-packages/sixer.py", line 1178, in patch
    new_content = operation.patch(content)
  File "/Users/deil/Library/Python/3.4/lib/python/site-packages/sixer.py", line 398, in patch
    new_content = self.patcher.add_import(new_content, 'from six.moves import range')
  File "/Users/deil/Library/Python/3.4/lib/python/site-packages/sixer.py", line 1153, in add_import
    return self.add_import_names(content, line, names)
  File "/Users/deil/Library/Python/3.4/lib/python/site-packages/sixer.py", line 1111, in add_import_names
    "third-party modules in %s" % import_groups)
Exception: Unable to locate the import group of third-party modules in [(937, 988, {'__future__', 'numpy'})]
```

Am I doing something wrong or is this a bug in sixer?

**Comment by vstinner** [comment]
Hi,

Thanks for the bug report. I knew that I should fix this code :-) Sixer is very strict, whereas a simple warning could be emitted.

This issue is now fixed by the change 080c6c3edc869294efeff29cd61b0d24a212acab in the development version, can you please try it?

I added numpy to known third-party libraries, and I added a command line option --third-party to specify other module names (separated by comma).

**Comment by cdeil** [comment]
The issue is fixed. Thanks!

**Comment by vstinner** [comment]
Thanks for your contribution! I released sixer 1.1.

Did sixer produce invalid changes? Are you happy of the result? :-)

**Comment by cdeil** [comment]
I ended up using `python-modernize` for what I wanted to do today.
Next time I need to do some sixing I'll try `sixer` ... thanks for making and maintaining it!
